index.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Jonathan Arias Gallego | Portfolio</title>
+</head>
+
+<body>
+  <header>
+    <nav id="navbar" class="open">
+      <div class="toolbar">
+        <a href="#" class="toolbar__logo">My Logo</a>
+        <a href="#" id="toolbar__button"><img src="Assets/Icon - Menu.svg" alt=""></a>
+        <div class="toolbar__menu">
+          <a href="#porfolio" class="">Portfolio</a>
+          <a href="#about" class="">About</a>
+          <a href="#contact" class="">Contact</a>
+          <a href="#" class="menu__button"><img src="Assets/msg-box.svg" alt=""></a>
+        </div>
+      </div>
+    </nav>
+  </header>
+  <main>
+    <section id="portfolio">
+      <h1>Hey There. I’m Alex.</h1>
+      <h2>I'm a Software Developer</h2>
+      <p class="portfolio__paragraph">I can help you build a product , feature or website Look through some of my
+        work and experience! If you
+        like what you see and have a project you need coded,
+        don’t hesitate to contact me.
+      </p>
+      <ul class="social-media">
+        <li><a href=""><img src="Assets/Github.svg" alt=""></a></li>
+        <li><a href=""><img src="Assets/Linkedin.svg" alt=""></a></li>
+        <li><a href=""><img src="Assets/Icon - Angelist.svg" alt=""></a></li>
+        <li><a href=""><img src="Assets/Twitter.svg" alt=""></a></li>
+        <li><a href=""><img src="Assets/Medium.svg" alt=""></a></li>
+      </ul>
+    </section>
+    <section id="experience">
+            <div class="experience__subtitle">
+                <h3>My Recent Works</h3>
+                <div class="experience__hr"></div>
+            </div>
+            <div class="experience__multi-post">
+                <img src="Assets/Img Placeholder.svg" alt="">
+                <div>
+                    <h4>Multi-Post Stories</h4>
+                    <p class="experience__multi-post-paragraph">A daily selection of privately personalized reads; no
+                        accounts or sign-ups required. has been the
+                        industry's standard dummy text ever since the 1500s, when an unknown printer took a standard
+                        dummy
+                        text.</p>
+                    <ul class="multi-post__stack">
+                        <li class="stack__css">CSS</li>
+                        <li class="stack__html">HTML</li>
+                        <li class="stack__bootstrap">Bootstrap</li>
+                        <li class="stack__ruby">Ruby</li>
+                    </ul>
+                    <button type="button" class="experience__button">See project</button>
+                </div>
```

Comment:
According to the project requirements, the work cards should be aligned using Flexbox in order to have more options for alignment. I advise you to use this in order to stick with the requirements

Reply:
I have created the experience__multi-post class to use the flexbox property in the section indicated by the code reviewer and thus comply with the requirement of the project in which you have to create a card with flexbox to organize its elements and use a grid to accommodate the rest of the cards.